src/com/konovalova/lesson9/Notepad.java

```diff
@@ -0,0 +1,52 @@
+package com.konovalova.lesson9;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Scanner;
+import java.util.Date;
+import java.util.HashMap;
+
+public class Notepad {
+
+    public Record[] notepad;
+    public Notepad(){}
+    public void addRecord(int id){
```

Comment:
I don't think that it's an id (more appropriate name is 'size', 'count', 'number' and ctr...